DESCRIPTION

```diff
@@ -11,8 +11,11 @@ Imports:
     pack,
     methods,
     stats
-SystemRequirements: Python (>= 2.7.0) with header files and shared library
-    (UNIX) / static library (Windows)
+SystemRequirements: 
+    Python (>= 2.7.0) with header files and shared library,
```

Comment:
Omit Python as a requirement since it's not a PRErequisite.  That is, PythonEmbedInR installs Python itself.